stdlib/urllib/request.pyi

```diff
@@ -227,6 +229,12 @@ class ftpwrapper:  # undocumented
     def __init__(
         self, user: str, passwd: str, host: str, port: int, dirs: str, timeout: Optional[float] = ..., persistent: bool = ...
     ) -> None: ...
+    def close(self) -> None: ...
+    def endtransfer(self) -> None: ...
+    def file_close(self) -> None: ...
+    def init(self) -> None: ...
+    def real_close(self) -> None: ...
+    def retrfile(self, file: str, type: str) -> None: ...
```

Comment:
`retrfile` returns a tuple, `return (ftpobj, retrlen)`. Please figure out the types of `ftpobj` and `retrlen` variables, and add the correct return type here instead of `None`.

Reply:
I didn't notice that sneaky [line](https://github.com/python/cpython/blob/a0bd9e9c11f5f52c7ddd19144c8230da016b53c6/Lib/urllib/request.py#L2464) because of the [blank line](https://github.com/python/cpython/blob/a0bd9e9c11f5f52c7ddd19144c8230da016b53c6/Lib/urllib/request.py#L2459).

Reply:
I ctrl+f `return` :)